app/services/flow/flows/provider_capital.rb

```diff
@@ -29,6 +33,17 @@ class ProviderCapital < FlowSteps
             application.uploading_bank_statements? ? :means_summaries : :income_summary
           end,
         },
+        identify_incomes: {
+          path: ->(application) { urls.providers_legal_aid_application_means_identify_incomes_path(application) },
+          forward: lambda do |application|
+            application.income_types? ? :cash_incomes : :student_finances
+          end,
+          check_answers: lambda do |application|
+            return :cash_incomes if application.income_types?
+
+            application.uploading_bank_statements? ? :means_summaries : :income_summary
```

Comment:
Just noticed this - we don't need this conditional as we only need to redirect to the  `:income_summary` page on the Truelayer route - and anyone coming from this page will be on the enhanced bank upload route. So we can just redirect straight to `:means_summaries`

Reply:
Presumably that's the case for the existing `:identify_types_of_income` flow step too?

If it is, let's open a separate PR to fix that!

Reply:
Yes so since the introduction of this page then the check answers redirect can be updated for that flow step too, since the pages are now separate! 

Separate PR is potentially a good idea to update the `:identify_types_of_income` step as we might need to update/remove tests too 👌